command/server/cipher_suites.go

```diff
@@ -0,0 +1,636 @@
+package server
```

Comment:
Please remove this file and use the code I pointed you to earlier. This is code copied explicitly without attribution from Go's source code and until/unless we get a legal expert to OK it then I don't want to take that chance.

Reply:
fixed